pkg/cluster/kubeclient/ready.go

```diff
@@ -68,6 +70,9 @@ func (u *kubeclient) WaitForReadySyncPod(ctx context.Context) error {
 					u.log.Info(err.ErrStatus.Details.Causes[0].Message)
 				}
 				err = nil
+			case utilerrors.IsMatchingSyscallError(err, syscall.ECONNRESET):
```

Comment:
have you seen this error occur?  please can you share a link?

Reply:
https://github.com/openshift/openshift-azure/issues/1400#issuecomment-483920988
(should be connection refused, i'll update)